### ISSUE TITLE
Improve the style of the website

### ISSUE THREAD

**Issue by zaikunzhang** [pdfo/pdfo.github.io#10]
Now, the style of the PDFO website seems a bit "old" to me. Let's improve it and make it look more "young"/modern. Maybe it is because of the colors. Let's explore and see.

**Comment by ragonneau** [comment]
Hi @zaikunzhang,

I am considering implementing the website using [Sphinx](https://www.sphinx-doc.org/en/master/). It would allow us to write Markdown files instead of heavy and unreadable HTML files. Please have a look [here](https://sphinx-themes.org) and let me know if you see any theme that keeps your attention.

Thank you,
Tom.

**Comment by zaikunzhang** [comment]
Good idea! You can choose a few themes and then we discuss them. I trust your taste.
